package.json

```diff
@@ -30,7 +30,8 @@
     "lint": "eslint .",
     "lint:fix": "eslint --fix .",
     "clean": "rm -rf test/coverage",
-    "version": "./changelog_assert_entries $npm_package_version"
+    "safefreeze": "rm -rf node_modules npm-shrinkwrap.json && npm install && npm dedupe && npm test && npm shrinkwrap --dev && touch package.json npm-shrinkwrap.json",
```

Comment:
@pdehaan another concern: I've used `npm shrinkwrap --dev` here as you suggested in #63, but I think it's rather `npm shrinkwrap` that should be used. Because if we use `npm shrinkwrap --dev` to generate the `npm-shrinkwrap.json` file, everyone doing `npm install convict` will also install the `devDependencies`. And the `devDependencies` are only useful for the contributors and maintainers of Convict. What do you think of that please?